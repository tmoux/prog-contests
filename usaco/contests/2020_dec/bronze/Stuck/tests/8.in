40
N 9593614 4133589
E 5828993 9544918
N 7376882 7844822
E 5326717 3124030
N 4311574 1530126
N 2781486 3550253
N 8803074 3327814
N 6478179 5138377
E 4415775 1727088
N 5935710 9806732
E 6011017 9400346
N 7871137 7745691
N 2423943 5122573
E 300925 449291
E 6894018 7277217
E 596963 2575055
E 2963858 1378130
E 698571 372661
N 3689685 7304789
N 5722604 5756851
N 6349250 1767868
N 1109485 9639006
N 5414639 2062949
E 80967 4880226
N 387677 4290596
N 4740865 4887560
E 127955 367770
E 1359211 1066342
E 245196 7272379
N 858270 2994984
E 3794427 9344234
E 8647283 2970072
E 1245830 901063
N 8966950 982030
N 9653714 3886059
E 134047 8626924
E 3189435 1271232
E 7622954 2630443
E 4945486 2875640
E 2089159 6250262
