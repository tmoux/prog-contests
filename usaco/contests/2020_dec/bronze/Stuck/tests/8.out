3143628
2042144
1555524
Infinity
4720136
2700009
3949403
4205857
4551175
Infinity
1860120
Infinity
1127689
Infinity
Infinity
3714611
6003092
Infinity
Infinity
3587383
1356162
Infinity
812691
2700519
589630
4456674
Infinity
7607739
4495669
1885242
3582455
Infinity
Infinity
1648413
3391158
3555638
5777515
Infinity
1403764
2651706
